libpyvinyl/Parameters/Collections.py

```diff
@@ -5,6 +5,38 @@
 
 from libpyvinyl.AbstractBaseClass import AbstractBaseClass
 from .Parameter import Parameter
+from pint.quantity import Quantity
+from pint.unit import Unit, UnitsContainer
```

Comment:
don't forget to add pint to the requirements (if not already there)

Reply:
It is already there.